### ISSUE TITLE
Navigation to a Blank Page in Class library project leads to Exception in UWP

### ISSUE THREAD

**Issue by syncsiva** [microsoft/microsoft-ui-xaml#4618]
**Describe the bug**
Navigation to a Blank Page in Class library project leads to Exception

**Steps to reproduce the bug**
[ReunionBlackAppUWP.zip](https://github.com/microsoft/ProjectReunion/files/6172314/ReunionBlackAppUWP.zip)

1. Just open the sample and run, you will get xaml parse exception
Exception thrown: 'Windows.UI.Xaml.Markup.XamlParseException' in ClassLibrary1.dll
An exception of type 'Windows.UI.Xaml.Markup.XamlParseException' occurred in ClassLibrary1.dll but was not handled in user code
XAML parsing failed.
2. Its just a blank app and a class library with blank winui page.

**Expected behavior**
Should work without any exception. It works fine with WinUI3 Preview4 and facing exception only after updating to Project reunion. 

**Screenshots**
![image](https://user-images.githubusercontent.com/18731572/111803906-884c5100-88f5-11eb-8b3f-ed4a438c4d86.png)

**Version Info**
Microsoft.ProjectReunion -> Version="0.5.0-prerelease
Visual Studio : VS 2019 16.10 Preview 1.0
Microsoft Windows Version 20H2 (OS build 19042.867)

**Additional context**
It works fine with WinUI3 Preview4 and facing exception only after updating to Project. Any workaround is very helpful.

**Comment by syncsiva** [comment]
Any workaround solution to the problem would be greatly appreciated!

**Comment by bpulliam** [comment]
UWP is no longer supported by WinUI 3.  If you can repro the issue on current builds without UWP, please reactivate or open a new issue.

Thanks!
